.kokoro

```diff
@@ -114,7 +114,7 @@ run_cocoapods() {
 
     scripts/prep_all
     scripts/build_all --verbose
-    if [ "$IS_RELEASE" -eq "1" ]; then
+    if [[ "$IS_RELEASE" && "$IS_RELEASE" -eq "1" ]]; then
```

Comment:
Not my favorite, since `IS_RELEASE=nonexistentVariable` will evaluate to true in `"$IS_RELEASE"`